### PR TITLE
no one ever tried this java stuff???

### PR DESCRIPTION
**Pull request by arris69**
(crashes if the model has a lot of meshes...)

### CODE REVIEW

File: port/jassimp/jassimp-native/src/jassimp.cpp [hunk 1]
```diff
@@ -5,12 +5,30 @@
...
+#define lprintf(...) __android_log_print(ANDROID_LOG_VERBOSE, __func__, __VA_ARGS__)
+#else
 #define lprintf(...) printf (__VA_ARGS__)
+#endif /* ANDROID */
 #else
 #define lprintf
 #endif
 
 
+static void dumpRefTable(JNIEnv *env){
```
Comment (jdduke) [inline port/jassimp/jassimp-native/src/jassimp.cpp]: I don't think we need this debug helper.

File: port/jassimp/jassimp-native/src/jassimp.cpp [hunk 2]
```diff
@@ -5,12 +5,30 @@
 
 
 #ifdef JNI_LOG
+#ifdef ANDROID
```
Comment (jdduke) [inline port/jassimp/jassimp-native/src/jassimp.cpp]: Not __ANDROID__?

**Comment by jdduke** [comment]
Thanks, I wonder how difficult it would be to get Travis to test compile this against the Android NDK.

File: port/jassimp/jassimp-native/src/jassimp.cpp (hunk 2 above)
Comment (acgessler) [inline port/jassimp/jassimp-native/src/jassimp.cpp]: \_\_ANDROID\_\_?

Comment (arris69) [inline port/jassimp/jassimp-native/src/jassimp.cpp]: looks like i'm not the only one...
http://stackoverflow.com/questions/15328751/android-macro-suddenly-not-defined

**Comment by acgessler** [comment]
Is this complete, i.e. can be merged or work in progress?

**Comment by arris69** [comment]
it can be merged, then this just fixed compile errors for android target.

**Comment by acgessler** [comment]
Ok, thanks - for further fixes, just open a new PR.

Also, if you want you could add jassimp to the travis CI build.

**Comment by arris69** [comment]
my question is how to do that?
http://yaml.travis-ci.org/ tells me that you can't add jdk: oracle_blabla to language: gcc
it's also not possible to use multiple languages (gcc java) at the same time.
2. how to install android ndk/sdk (normally you have to accept some braindead goole license also on commandline install), its ok for the assimp team if i provide a shellscript that install it without interaction?
3. guess it would makes sense to provide an android sample app (at least what dumps some info about the model file)?

regards

**Pull request merged**